models/object_detection_yolox/demo.py

```diff
@@ -0,0 +1,132 @@
+import cv2
+import numpy as np
+import argparse
+import time
+import moviepy.video.io.ImageSequenceClip
+from YoloX import YoloX
+
+backends = [cv2.dnn.DNN_BACKEND_OPENCV, cv2.dnn.DNN_BACKEND_CUDA]
+targets = [cv2.dnn.DNN_TARGET_CPU, cv2.dnn.DNN_TARGET_CUDA, cv2.dnn.DNN_TARGET_CUDA_FP16]
+help_msg_backends = "Choose one of the computation backends: {:d}: OpenCV implementation (default); {:d}: CUDA"
+help_msg_targets = "Chose one of the target computation devices: {:d}: CPU (default); {:d}: CUDA; {:d}: CUDA fp16"
+
+try:
+    backends += [cv2.dnn.DNN_BACKEND_TIMVX]
+    targets += [cv2.dnn.DNN_TARGET_NPU]
+    help_msg_backends += "; {:d}: TIMVX"
+    help_msg_targets += "; {:d}: NPU"
+except:
+    print('This version of OpenCV does not support TIM-VX and NPU. Visit https://gist.github.com/Sidd1609/5bb321c8733110ed613ec120c7c02e41 for more information.')
+
+with open('coco.names', 'rt') as f:
+    classes = f.read().rstrip('\n').split('\n')
```

Comment:
How about adding a member function in YoloX which returns the classes for the detetection results?

Reply:
replaced it with classes in demo instead of a separate member function

Reply:
Just added it as len=80 in YoloX.py